tests/standardTFReflectivityCalculation/testReflectivityCalculations.m

```diff
@@ -191,22 +197,27 @@ function testReflectivityCalculation(testCase, whichParallel, useCompiled)
             % contrasts) versions of the calculation, using both the MATLAB
             % and comiled (MEX) versions of each.
             
+            if useCompiled
+                % Skip the mex tests if coder is not installed. These tests will be marked as incomplete
+                testCase.assumeEqual(license('test', 'MATLAB_Coder'), 1, 'MATLAB Coder is not installed')
+            end
+
             [problem, result] = reflectivityCalculationTestingWrapper(testCase.problemDef, testCase.problemDefCells, testCase.problemDefLimits, testCase.controls, useCompiled, whichParallel);
 
-            testCase.verifyEqual(problem, testCase.expectedProblem, 'RelTol', testCase.tolerance, 'AbsTol', testCase.abs_tolerance);
-            testCase.verifyEqual(result, testCase.expectedResult, 'RelTol', testCase.tolerance, 'AbsTol', testCase.abs_tolerance);
+            testCase.verifyEqual(problem, testCase.expectedProblem, 'RelTol', testCase.tolerance, 'AbsTol', testCase.absTolerance);
+            testCase.verifyEqual(result, testCase.expectedResult, 'RelTol', testCase.tolerance, 'AbsTol', testCase.absTolerance);
```

Comment:
Could we also get rid of `reflectivityCalculationTestingWrapper` at this point by splitting this test into two, testing `reflectivityCalculation` and `reflectivityCalculation_mex` (and still parameterising over "whichParallel")?